ProcessesApi/V1/Gateways/SoleToJointGateway.cs

```diff
@@ -51,101 +53,94 @@ private async Task<Person> GetPersonById(Guid id)
+            _logger.LogDebug($"Calling Income API for payment agreement with tenancy ref: {tenancyRef}");
+            var route = $"{_apiGateway.ApiRoute}/agreements/{tenancyRef}";
+            return await _apiGateway.GetByIdAsync<PaymentAgreements>(route, tenancyRef, correlationId);
         }
 
         [LogCall]
-        private async Task<Tenancy> GetTenancyByReference(string tenureRef, Guid correlationId)
+        private async Task<Tenancy> GetTenancyByReference(string tenancyRef, Guid correlationId)
         {
-            _logger.LogDebug($"Calling Income API with tenancy ref: {tenureRef}");
-            var route = $"{_apiGateway.ApiRoute}/tenancies/{tenureRef}";
-            return await _apiGateway.GetByIdAsync<Tenancy>(route, tenureRef, correlationId);
+            _logger.LogDebug($"Calling Income API with tenancy ref: {tenancyRef}");
+            var route = $"{_apiGateway.ApiRoute}/tenancies/{tenancyRef}";
+            return await _apiGateway.GetByIdAsync<Tenancy>(route, tenancyRef, correlationId);
         }
 
-        public async Task<bool> CheckTenureFinanceRecords(string tenureRef)
+        private bool BR2(Guid tenantId, TenureInformation tenure)
         {
-            var paymentAgreements = await GetPaymentAgreementsByTenancyReference(tenureRef, Guid.NewGuid()).ConfigureAwait(false); // TODO: Confirm what correlation ID to use
-            if (paymentAgreements != null && paymentAgreements.Agreements.Count(x => x.Amount > 0) > 0)
-            {
-                return false;
-            }
-            else
-            {
-                var tenancy = await GetTenancyByReference(tenureRef, Guid.NewGuid()).ConfigureAwait(false); // TODO: Confirm what correlation ID to use
-                if (tenancy is null)
-                    return true;
+            var currentTenantDetails = tenure.HouseholdMembers.FirstOrDefault(x => x.Id == tenantId);
+            return currentTenantDetails.PersonTenureType == PersonTenureType.Tenant;
+        }
 
-                if (tenancy.nosp.active)
-                    return false;
+        private bool BR3(TenureInformation tenure)
+        {
+            return tenure.HouseholdMembers.Count(x => x.IsResponsible) <= 1;
+        }
 
-                return true;
-            }
+        private bool BR4(TenureInformation tenure)
+        {
+            return tenure.TenureType.Code == TenureTypes.Secure.Code;
         }
 
-        private string GetLegacyTagRef(TenureInformation tenure)
+        private bool BR6(TenureInformation tenure)
         {
-            var reference = tenure.LegacyReferences.FirstOrDefault(x => x.Name == "uh_tag_ref");
-            return reference?.Value;
+            return tenure.IsActive;
         }
 
-        public async Task<bool> CheckPersonTenureRecord(Guid tenureId, Guid proposedTenantId)
+        private async Task<bool> BR7(TenureInformation tenure)
```

Comment:
These check predicates look great to me.

The only thing I would change is to add short BR description from google sheet to each predicate as plain comment or xml comment, to help future readers understand what each BR means